rhel8/profiles/cis.profile

```diff
@@ -0,0 +1,869 @@
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5249
+
+    #### 3.4.3.8 Ensure nftables rules are permanent (Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5250
+
+    ### 3.4.4 Configure iptables
+
+    #### 3.4.4.1 Configure IPv4 iptables
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5251
+
+    ##### 3.4.4.1.1 Ensure default deny firewall policy (Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5252
+
+    ##### 3.4.4.1.2 Ensure loopback traffic is configured (Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5253
+
+    ##### 3.4.4.1.3 Ensure outbound and established connections are
+    #####           configured (Not Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5254
+
+    ##### 3.4.4.1.4 Ensure firewall rules exist for all open ports (Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5255
+
+    #### 3.4.4.2 Configure IPv6 ip6tables
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5256
+
+    ##### 3.4.4.2.1 Ensure IPv6 default deny firewall policy (Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5257
+
+    ##### 3.4.4.2.2 Ensure IPv6 loopback traffic is configured (Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5258
+
+    ##### 3.4.4.2.3 Ensure IPv6 outbound and established connections are
+    #####           configured (Not Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5260
+
+    ## 3.5 Ensure wireless interfaces are disabled (Scored)
+    - wireless_disable_interfaces
+
+    ## 3.6 Disable IPv6 (Not Scored)
+    - kernel_module_ipv6_option_disabled
+
+    # Logging and Auditing
+
+    ## 4.1 Configure System Accounting (auditd)
+
+    ### 4.1.1 Ensure auditing is enabled
+
+    #### 4.1.1.1 Ensure auditd is installed (Scored)
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5261
+
+    #### 4.1.1.2 Ensure auditd service is enabled (Scored)
+    - service_auditd_enabled
+
+    #### 4.1.1.3 Ensure auditing for processes that start prior to audit
+    ####         is enabled (Scored)
+    - grub2_audit_argument
+
+    #### 4.1.1.4 Ensure audit_backlog_limit is sufficient (Scored)
+    - grub2_audit_backlog_limit_argument
+
+    ### 4.1.2 Configure Data Retention
+
+    #### 4.1.2.1 Ensure audit log storage size is configured (Scored)
+    - auditd_data_retention_max_log_file
+
+    #### 4.1.2.2 Ensure audit logs are not automatically deleted (Scored)
+    - auditd_data_retention_max_log_file_action
+
+    #### 4.1.2.3 Ensure system is disabled when audit logs are full (Scored)
+    ##### space_left_action = email
+    - auditd_data_retention_admin_space_left_action
+    # TO DO: SET TO EMAIL
+
+    ##### action_mail_acct = root
+    - var_auditd_action_mail_acct = root
```

Comment:
```suggestion
    - var_auditd_action_mail_acct=root
```
The syntax for variables is `var_name=value`. The build system cannot process spaces in this case.

Reply:
ack & fixed